docs/src/api/class-tracing.md

```diff
@@ -126,11 +126,24 @@ a timeline preview.
 
 Whether to capture DOM snapshot on every action.
 
+### option: Tracing.start.video
+- `video` <[boolean]|[Object]>
+  - `width` <[int]> Video frame width.
+  - `height` <[int]> Video frame height.
+
+Whether to record video for all pages.
+
+Alternatively, specifies the dimensions of the recorded video. Actual picture of each page will be scaled down if necessary to fit the specified size.
+
 ## async method: Tracing.stop
+- returns: <[Object]>
+  - `videoFiles` <[Array]<[string]>> The list of video files created next to the exported trace file.
```

Comment:
I anticipate users asking how to map the file names to the pages, do we have an answer?

Reply:
We don't have the answer, but the intended use is for "artifacts for my test", where there is no such question. If anything comes up, we can return more fields here like `Video` objects.